pulsechain/subclients/blocks.py

```
@@ -0,0 +1,75 @@
+        super().__init__(subpath="blocks", request_handler=request_handler)
+
+    @paginated
+    def get_blocks(
+        self, block_type: list[str] | None = None, params: dict | None = None
+    ) -> tuple[BaseResponse, dict]:
+        """
+        Get information about blocks.
+        :return: BaseResponse containing information about the blocks.
+        """
+        if params:
+            if block_type:
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)